core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/ApacheSender42.java

```diff
@@ -66,16 +73,26 @@ public void dispose(HttpResponse response) {
 
     @Override
     public void close() {
-        try {
-            cm.shutdown();
-        } catch (Exception e) {
-            // chomp
+        synchronized (lock) {
```

Comment:
curious why we capture exception in the ApacheSender43 class but not here. Would like to learn more.

Reply:
The implementation of the shutdown method cannot throw

Reply:
but `close()` can